aequilibrae/paths/AoN.pyx

```diff
@@ -87,6 +92,25 @@ def one_to_all(origin, matrix, graph, result, aux_result, curr_thread):
     cdef double [:, :] node_load_view = aux_result.temp_node_loads[:, :, curr_thread]
     cdef long long [:] b_nodes_view = aux_result.temp_b_nodes[:, curr_thread]
 
+    # path saving file paths
+    cdef string path_file_base
+    cdef string path_index_file_base
+    cdef bool save_paths = False
+    cdef bool write_feather = True
+    if result.save_path_file == True:
```

Comment:
if True == True: ??

Reply:
result.save_path_file is a variable that defaults to false

Reply:
did you confuse it with save_paths just above, or did I miss your point?

Reply:
also, save_paths does not seem to be used anymore, so should be removed

Reply:
missed the point i think

```suggestion
    if result.save_path_file:
```

Reply:
indeed I did, didn't even notice :)